sci-physics/fastjet-contrib/fastjet-contrib-1.049.ebuild

```diff
@@ -22,12 +22,14 @@ RDEPEND="${DEPEND}"
 
 PATCHES=(
 	"${FILESDIR}"/${P}-soname.patch
+	"${FILESDIR}"/${P}-ar.patch
 )
 
 src_configure() {
-	CXX=`tc-getCXX`
-	./configure --prefix=/usr --fastjet-config=/usr/bin/fastjet-config CXX="${CXX}" CFLAGS="${CFLAGS}" CXXFLAGS="${CXXFLAGS}" FFLAGS="${FFLAGS}" LDFLAGS="${LDFLAGS}" || die
+	tc-export CXX AR RANLIB
+	./configure --prefix=/usr --fastjet-config=/usr/bin/fastjet-config RANLIB="${RANLIB}" AR="${AR}" CXX="${CXX}" CFLAGS="${CFLAGS}" CXXFLAGS="${CXXFLAGS}" FFLAGS="${FFLAGS}" LDFLAGS="${LDFLAGS}" || die
```

Comment:
I think it is a good idea to revision bump the ebuild to -r1. I'll do this while merging.